modules/monitor-osp-disable-helm.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// configuration/monitor-acs.adoc
+:_content-type: PROCEDURE
+[id="monitor-osp-disable-helm_{context}"]
+= Disabling {osp} monitoring for Central services by using Helm
+
+To disable the default monitoring by using Helm, change the configuration options in the `central-services` Helm chart. For more information on configuration options, see the documents in the "Additional resources" section.
```

Comment:
If we have section id then we can add a link to it
```suggestion
To disable the default monitoring by using Helm, change the configuration options in the `central-services` Helm chart. For more information on configuration options, see the documents in the xref:additional-resources_{context}[Additional resources]  section.
```

Reply:
Unfortunately no, will break when publishing to Customer Portal :(